stable/dask/templates/dask-worker-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "dask.worker-fullname" . }}
+  labels:
+    app: {{ template "dask.name" . }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    component: "{{ .Release.Name }}-{{ .Values.worker.component }}"
+spec:
+  replicas: {{ .Values.worker.replicas }}
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: {{ template "dask.name" . }}
+        release: {{ .Release.Name | quote }}
+        component: "{{ .Release.Name }}-{{ .Values.worker.component }}"
+    spec:
+      containers:
+        - name: {{ template "dask.worker-fullname" . }}
+          image: 
+            repository: {{ .Values.worker.image | quote }}
+            tag: {{ .Values.worker.imageTag | quote }}
+            pullPolicy: IfNotPresent
+          args: ["dask-worker",
```

Comment:
Just a suggestion for a consistent list style:

```yaml
          args: 
            - dask-worker
            - '{{ template "dask.scheduler-fullname" . }}:{{ .Values.scheduler.servicePort }}'
          {{- if .Values.worker.limits -}}
            - --nthreads
            - {{ default .Values.worker.limits.cpu 1 | quote }}
            - --memory-limit
            - {{ default .Values.worker.limits.memory "4GiB" | quote }}
          {{- end -}}
            - --no-bokeh
```
And don't use the `default` function here. Add the defaults to `values.yaml`.